src/components/game/Game.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import './Game.css';
+import ReactDOM from 'react-dom';
 import Board from '../board/Board';
 import { addTroggle, moveTroggles } from '../troggle/Troggle';
 import Status from '../status/Status';
+import Quit from '../quit/Quit';
+// eslint-disable-next-line import/no-cycle
+import Menu from '../menu/Menu';
```

Comment:
Codacy has a fix for the issue: [Replace `Menu·from·'../menu/Menu` with `'./Game.css`](https://app.codacy.com/gh/msaperst/number-munchers/pullRequest?prid=8124485)
```suggestion
import './Game.css';
```